src/app/pages/app-bar/page-wide-search/page-wide-search.component.html

```diff
@@ -0,0 +1,70 @@
+<mat-toolbar data-cy="pxb-toolbar" color="primary" class="main-header mat-elevation-z4 t" [class.hidden]="toggleSearch">
+    <mat-toolbar-row>
+        <button data-cy="toolbar-menu" mat-icon-button *ngIf="isSmall" (click)="toggleMenu()" style="margin-left: -8px">
+            <mat-icon>menu</mat-icon>
+        </button>
+        <div class="mat-title" [style.marginLeft.px]="isSmall ? 24 : 0">Page Search</div>
+        <pxb-spacer></pxb-spacer>
+
+        <div pxb-actions class="appbar-actions">
+            <button mat-icon-button>
+                <mat-icon style="margin: 0 8px">help</mat-icon>
+            </button>
+            <button mat-icon-button>
+                <mat-icon style="margin: 0 8px" [matBadge]="88" matBadgeColor="warn">notifications</mat-icon>
+            </button>
```

Comment:
`0 4px` margin works better for these icons

Reply:
Ok you got it.